customer-service-tests/IncidentTypeFactoryTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+using customer_service;
+using customer_service.Models;
+
+namespace customer_service_tests
+{
+    public class IncidentTypeFactoryTests
+    {
+        [Fact]
+        public void CanInstantiateIncidentTypeFactory()
+        {
+            var fact = IncidentTypeFactory.Instance;
+            Assert.NotNull(fact);
+        }
+
+        [Fact]
+        public void IncidentTypeFactoryIsSingleton()
+        {
+            var fact1 = IncidentTypeFactory.Instance;
+            var fact2 = IncidentTypeFactory.Instance;
+            Assert.Equal(fact1, fact2);
+        }
+
+        [Fact]
+        public void CanGetAllIncidentTypes()
```

Comment:
Little fuzzy on the name here, you call it "get all" but then only test for the first one in the list.  For a get all, I'd expect something more like where you check the length of the list to equal a length you expect it to have, or check that all the Labels are the labels you expect.